src/config/example.toml

```diff
@@ -61,6 +61,10 @@ in_flight_barrier_nums = 10000
 async_stack_trace = "ReleaseVerbose"
 unique_user_stream_errors = 10
 
+[streaming.auto_dump_heap_profile]
```

Comment:
Emmm I think it's more suitable to be placed on `[server]` section

Reply:
Hmmm, one problem is we might want set different profile configs for cn/meta/compactor. We might need config for different servers like `[compute]/[compactor]`?